gen/sympa

```diff
@@ -57,7 +51,7 @@ for my $listName (keys %$mailinglistStruc) {
     open FILE,">$fileName" or die "Cannot open $fileName: $! \n";
 
     for my $mail (keys %{$mailinglistStruc->{$listName}}) {
```

Comment:
I think we could add `sort` here to have unified output just like in the `symba_cesnet`.

Reply:
added